1-callbacks/solution.js

```diff
@@ -33,15 +33,41 @@ node solution.js name1 name2 name3
 */
 
 function solution() {
-    // YOUR SOLUTION GOES HERE
-
-    // you get your 5 names here
-
-    // iterate the names array and validate them with the method
-
-    // log the final result
+  // YOUR SOLUTION GOES HERE
+
+  // you get your 5 names here
+
+  // Using an array:
+  // const users = ["Mary", "Daniel", "Stacy", "Gerardo", "Diego"];
+  
+  // Using process.argv property:
+  const { argv } = require('process');
+  const users = argv.filter((item, index) => index > 1);
```

Comment:
i think a straightforwardto achive the same result is using slice, since we always should skip the first 2 args